dojo/models.py

```diff
@@ -1226,7 +1226,7 @@ def __str__(self):
 
 
 class Finding(models.Model):
-    title = models.CharField(max_length=511)
+    title = models.TextField(max_length=511)
```

Comment:
I think the type was changed to CharField for a reason (index?) so unless there is a good reason, it should stay CharField?